### PR TITLE
Add PDM support for Python projects

### PR DESCRIPTION
**Pull request by buurro**
This PR adds support for PDM package manager, as suggested in #877

### CODE REVIEW

**Comment by awnion** [comment]
+1 for PDM support

File: docs/pages/docs/providers/python.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ If `pyproject.toml` (w/ `poetry.lock`)
 poetry install --no-dev --no-interactive --no-ansi
 ```
 
+If `pyproject.toml` (w/ `pdm.lock`)
+
+```
+pdm install
```
Comment (patrick91) [inline docs/pages/docs/providers/python.md:54]:
```suggestion
pdm install --prod
```

Comment (patrick91) [reply docs/pages/docs/providers/python.md:54]: same below :)

**Comment by patrick91** [comment]
@frostming does this look good to you? (hope you don't mind the ping 😊)

File: docs/pages/docs/providers/python.md (hunk 1 above)
Comment (buurro) [reply docs/pages/docs/providers/python.md:54]: makes sense

**Review by coffee-cup — APPROVED** [review]
Thanks 🎉

**Comment by patrick91** [comment]
@coffee-cup are these immediately supported on railway? 😊

**Pull request merged**